scoreCard.js

```diff
@@ -0,0 +1,53 @@
+class ScoreCard {
```

Comment:
Overall this is really nice - the logic is tight and the code is easy to understand. Likewise your tests are comprehensive and easy to follow. Good job!